src/domain/mod.rs

```diff
@@ -132,9 +132,17 @@ where
     ) -> Result<(), Error> {
         // We need to install a probe on source-fed attributes in
         // order to determine their progress.
-        let probed_pairs = pairs.probe_with(&mut self.input_probe);
 
-        self.create_attribute(name, config, &probed_pairs)?;
+        // We do not want to probe timeless attributes.
+        // Sources of timeless attributes either are not able to or do not
+        // want to provide valid domain timestamps.
```

Comment:
One more sentence here please, on why forcing them to specify times is problematic (because they might stall everyone else).